noxfile.py

```diff
@@ -38,7 +38,7 @@ def cover(session):
 
 @nox.session(py=DEFAULT_INTERPRETER)
 def lint(session):
-    session.install("black", "flake8")
+    session.install("black", "flake8", "flake8-pyproject")
```

Comment:
Apparently the maintainers of flake8 don't believe in pyproject.toml, so this plugin is required for now.

Reply:
Wild?